Sources/Slipstream/W3C/Elements/EmbeddedContent/SVG/SVGCircle.swift

```diff
@@ -0,0 +1,48 @@
+import SwiftSoup
+
+/// A view that represents an SVG circle element.
+///
+/// SVGCircle creates a circular shape within an SVG context.
+///
+/// ```swift
+/// struct MySiteContent: View {
+///   var body: some View {
+///     Body {
+///       SVG(viewBox: "0 0 100 100") {
+///         SVGCircle(cx: "50", cy: "50", r: "40")
```

Comment:
Ideally we use more idiomatic Swift names for the arguments. E.g. `x: CGFloat, y: CGFloat, radius: CGFloat`, and maybe `origin: CGPoint, radius: CGFloat`.